### PR TITLE
Mailgun improvements

### PR DESCRIPTION
**Pull request by roosnic1**

### CODE REVIEW

File: packages/api/src/mails/MailgunMailProvider.ts [hunk 1]
```diff
@@ -88,21 +90,27 @@ export class MailgunMailProvider extends BaseMailProvider {
...
+        {
+          host: this.baseDomain,
+          path: `/v3/${this.mailDomain}/messages`,
+          method: 'POST',
+          headers: {
+            Accept: 'application/json',
+            Authorization: `Basic ${this.auth}`
+          }
+        },
+        (err, res) => {
```
Comment (0815pascal) [inline packages/api/src/mails/MailgunMailProvider.ts:110]: where are you using `res` ?

**Review by 0815pascal — CHANGES_REQUESTED** [review]
It looks good, just one question

**Review by penk85 — DISMISSED** [review]

File: packages/api/src/mails/MailgunMailProvider.ts (hunk 1 above)
Comment (roosnic1) [reply packages/api/src/mails/MailgunMailProvider.ts:110]: Good point. Fixed it.

**Review by bishoymelek — APPROVED** [review]

**Review by penk85 — APPROVED** [review]

**Review by bishoymelek — APPROVED** [review]

**Review by faebebin — APPROVED** [review]
Looks good

**Review by 0815pascal — APPROVED** [review]

**Pull request merged**